applications/StructuralMechanicsApplication/tests/cpp_tests/test_solid_transient_sensitivity.cpp

```diff
@@ -134,7 +134,7 @@ SolvingStrategyType::Pointer CreatePrimalSolvingStrategy(ModelPart* pModelPart)
     SchemeType::Pointer p_scheme =
         Kratos::make_shared<ResidualBasedBossakDisplacementScheme<SparseSpaceType, LocalSpaceType>>(0.0);
     ConvergenceCriteriaType::Pointer p_conv_criteria =
-        Kratos::make_shared<ResidualCriteria<SparseSpaceType, LocalSpaceType>>(1e-24, 1e-25);
+        Kratos::make_shared<ResidualCriteria<SparseSpaceType, LocalSpaceType>>(1e-12, 1e-14);
```

Comment:
why do you decrease the tolerance 

FYI @msandre

Reply:
Run the current tests with the current tolerance, it doesn't converge. With this tolerance it converges and the test pass